### PR TITLE
fix: replace hard coded bx prefix with variable

### PR DESCRIPTION
**Pull request by alisonjoseph**
Noticed some hard coded `.bx--` classnames. Did a find on the entire project and replaced them with `#{$prefix}--`

### CODE REVIEW

**Review by asudoh — APPROVED** [review]
Thanks @alisonjoseph for spotting these!!

File: src/components/combo-box/_combo-box.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@
 @import '../list-box/list-box';
 
 @include exports('combo-box') {
-  .bx--combo-box > .bx--list-box__field {
+  .#{$prefix}--combo-box > .#{$prefix}--list-box__field {
```
Comment (tw15egan) [inline src/components/combo-box/_combo-box.scss:14]: Somewhat related to your PR, but I noticed that `ComboBox` has an extra `box-shadow` that could be bundled in with these changes 

screen shot 2018-11-02 at 10 24 15 am

File: src/components/multi-select/_multi-select.scss [hunk 2]
```diff
@@ -13,11 +13,11 @@
 @import '../list-box/list-box';
 
 @include exports('multi-select') {
-  .bx--multi-select.bx--combo-box > .bx--list-box__field {
+  .#{$prefix}--multi-select.#{$prefix}--combo-box > .#{$prefix}--list-box__field {
```
Comment (tw15egan) [inline src/components/multi-select/_multi-select.scss:16]: I'm also noticing that box-shadow here, but it looks like we never added the `gray-50` border bottom. Perhaps this is a bigger change than I thought

File: src/components/overflow-menu/_overflow-menu.scss [hunk 3]
```diff
@@ -145,7 +145,7 @@
   }
 
   .#{$prefix}--overflow-menu-options__option--danger {
-    .bx--overflow-menu-options__btn {
+    .#{$prefix}--overflow-menu-options__btn {
```
Comment (tw15egan) [inline src/components/overflow-menu/_overflow-menu.scss:148]: I'm not sure if this has always been the case, but the overflow-menu doesn't show up in experimental mode

**Review by tw15egan — APPROVED** [review]
Small comments that can be addressed quickly here or another PR, but other than that looks great! Thanks for catching these 👍

File: src/components/overflow-menu/_overflow-menu.scss (hunk 3 above)
Comment (alisonjoseph) [reply src/components/overflow-menu/_overflow-menu.scss:148]: Looks like this is related to product switcher? We can fix this when we build out the experimental overflow menu.
screen shot 2018-11-02 at 10 47 45 am

Comment (tw15egan) [reply src/components/overflow-menu/_overflow-menu.scss:148]: Yeah I can work on that one next week and get it figured out 👍

File: src/components/combo-box/_combo-box.scss (hunk 1 above)
Comment (alisonjoseph) [reply src/components/combo-box/_combo-box.scss:14]: Looks like there is box shadow and also the border-bottom on the input inside the list box? 🤔

File: src/components/multi-select/_multi-select.scss (hunk 2 above)
Comment (alisonjoseph) [reply src/components/multi-select/_multi-select.scss:16]: Might be better in a separate issue around list box styles, there is a lot going on here 🙃

Comment (tw15egan) [reply src/components/multi-select/_multi-select.scss:16]: Yeah, I'll work on that this morning and fix that up. Disregard!

**Pull request merged**